sentry-android/src/main/java/com/joshdholtz/sentry/Sentry.java

```diff
@@ -127,6 +132,7 @@ public static void init(Context context, String dsn, boolean setupUncaughtExcept
 		Sentry.getInstance().dsn = dsn;
 		Sentry.getInstance().packageName = context.getPackageName();
 		Sentry.getInstance().verifySsl = getVerifySsl(dsn);
+		Sentry.getInstance().contexts = ReadContexts(Sentry.getInstance().context);
```

Comment:
The new code runs at `init` time, which is most likely app-launch time. It could be better to do this work lazily. Perhaps we could build the `contexts` map when the first event is created.

I wouldn't want to go doing to much work in the uncaught exception handler either though.

Thoughts?